File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    let anOddNum = [];
    let anEvenNum = [];
    
    for(let i = 0; i < num_list.length; i++){
        if(num_list[i] % 2 == 0){
            anEvenNum.push(num_list[i])
        }else{
            anOddNum.push(num_list[i])
        }
    }
    answer.push(anEvenNum.length);
    answer.push(anOddNum.length);
    
    return answer;
}

File: 프로그래머스/unrated/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    a = []
    b = []
    for i in num_list:
        if i % 2 == 0:
            a.append(i)
        else:
            b.append(i)
    answer.append(len(a))
    answer.append(len(b))
    
    return answer